docker/wsgi.py

```diff
@@ -0,0 +1,70 @@
+from mms import mxnet_model_server
+import sys
+import json
+import os
+from mms.arg_parser import ArgParser
+
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+mms_arg_header = 'MMS Argument'
+args = []
+found_mms_args = 0
+config_file = os.environ['MXNET_MODEL_SERVER_CONFIG']
+is_gpu_image = os.environ['GPU_IMAGE']
+
+
+def read_models_from_file():
+    """
+    This method reads the model's meta-data from a local file. This is used for MMS in
+    :return: model's meta-data
+    """
+    models = json.load(open("/mxnet_model_server/.models"))
+    return models
+
+
+try:
+    f = open(config_file)
```

Comment:
can we please use more descriptive naming? i.e. config_file instead of f and config_file_path instead of config_file